src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -872,19 +907,17 @@ private static bool TryGetEasyRequestFromContext(IntPtr context, out EasyRequest
                 return false;
             }
 
-            [Conditional(VerboseDebuggingConditional)]
-            private void VerboseTrace(string text = null, [CallerMemberName] string memberName = null, EasyRequest easy = null)
+            private void EventSourceTrace<TArg0>(string formatMessage, TArg0 arg0, EasyRequest easy = null, [CallerMemberName] string memberName = null)
             {
-                CurlHandler.VerboseTrace(text, memberName, easy, agent: this);
+                if (EventSourceTracingEnabled)
+                {
+                    EventSourceTrace(string.Format(formatMessage, arg0), easy, memberName);
+                }
             }
 
-            [Conditional(VerboseDebuggingConditional)]
-            private void VerboseTraceIf(bool condition, string text = null, [CallerMemberName] string memberName = null, EasyRequest easy = null)
+            private void EventSourceTrace(string message = null, EasyRequest easy = null, [CallerMemberName] string memberName = null)
             {
-                if (condition)
-                {
-                    CurlHandler.VerboseTrace(text, memberName, easy, agent: this);
-                }
+                CurlHandler.EventSourceTrace(message: message, agent: this, easy: easy, memberName: memberName);
```

Comment:
Should there be a check for 'if (EventSourceTracingEnabled)'? Or is it assumed that all callers know to go through the other overloads to this method?

Maybe renaming this method to something like "EventSourceTracePrivate" would ensure no one calls this method directly.